web/modules/custom/usagov_ssg_postprocessing/src/EventSubscriber/TomeEventSubscriber.php

```diff
@@ -233,6 +233,9 @@ public function addAgencyIndexes(CollectPathsEvent $event): void {
 
   }
 
+  /**
+   * @return array<mixed>
```

Comment:
Function returns an array of strings, so this should be:

```
```
/**
 * @return string[]
 */
```